common/metrics/defs.go

```diff
@@ -713,6 +726,13 @@ var ScopeDefs = map[ServiceIdx]map[int]scopeDefinition{
 		PersistenceCompleteForkBranchScope:                       {operation: "CompleteForkBranch", tags: map[string]string{ShardTagName: NoneShardsTagValue}},
 		PersistenceGetHistoryTreeScope:                           {operation: "GetHistoryTree", tags: map[string]string{ShardTagName: NoneShardsTagValue}},
 
+		BlobstoreClientUploadScope:         {operation: "Upload"},
```

Comment:
Can you also include CadenceRoleTag for all blob client operations.  This makes it easier to just create a dashboard for all blob store operations.